### PR TITLE
hover and tooltip for button

### PR DESCRIPTION
**Pull request by gintama91**
### Description
hover n tooltip

### Image(if needed, helps for a faster review)

Screenshot_20231019_125204

### Checklist

- [ ] Run tests locally
- [ ] Run linter(check for linter errors)

### CODE REVIEW

**Comment by gintama91** [comment]
i cant seem to screenshot tooltip so thats not visible.

File: lacci/lib/shoes/drawables/button.rb [hunk 1]
```diff
@@ -22,12 +22,21 @@ def initialize(text, width: nil, height: nil, top: nil, left: nil, color: nil, p
         @block&.call
       end
 
+      bind_self_event("hover") do
+        @log.debug("Button hovered, calling handler") if @block
```
Comment (noahgibbs) [inline lacci/lib/shoes/drawables/button.rb:26]: The debug should probably not stay in the final version.

**Comment by noahgibbs** [comment]
Other than the one thing with the log debug, looks good to me. Approved.

**Comment by gintama91** [comment]
removed log debug

File: lacci/lib/shoes/drawables/button.rb (hunk 1 above)
Comment (gintama91) [reply lacci/lib/shoes/drawables/button.rb]: hm any reason for that? when we are doing log,debug for click? and why not for hover?

Comment (gintama91) [reply lacci/lib/shoes/drawables/button.rb]: ooo okay

**Pull request merged**